src/test/java/org/folio/search/controller/IndexControllerTest.java

```diff
@@ -52,6 +58,102 @@ void createIndex_positive() throws Exception {
       .andExpect(jsonPath("$.indices", is(List.of(INDEX_NAME))));
   }
 
+  @Test
+  void createIndex_negative_indexAlreadyExists() throws Exception {
+    var requestBuilder = post("/search/index/indices")
```

Comment:
Could you extract it to a constant or put it to `org.folio.search.support.base.ApiEndpoints` class, so that it can be reused?